### PR TITLE
Enabling async server tree in insiders

### PR DESCRIPTION
**Pull request by aasimkhan30**
This PR fixes #

### CODE REVIEW

File: src/sql/workbench/services/objectExplorer/browser/treeCreationUtils.ts
Comment (Charles-Gagnon) [inline src/sql/workbench/services/objectExplorer/browser/treeCreationUtils.ts:145]: If someone has preview features disabled it feels like we should still be ignoring this setting.

Comment (aasimkhan30) [reply src/sql/workbench/services/objectExplorer/browser/treeCreationUtils.ts:145]: The original idea is to make this the main tree in the next release. I worry if we hide it behind the preview flag, we might reduce the number of people testing it in insiders.

Comment (Charles-Gagnon) [reply src/sql/workbench/services/objectExplorer/browser/treeCreationUtils.ts:145]: That plan seems fine to me (I assume you'll be making sure we have plenty of people testing it) - but if you're going that route I'd remove the (Preview) text from the setting description then. I just don't think we should have any "preview" functionality that isn't dependent on the preview features flag as well - but in this case it seems like less of preview functionality and more like just a general improvement to the core feature.

File: resources/localization/LCL/de/sql.xlf.lcl
Comment (Charles-Gagnon) [inline resources/localization/LCL/de/sql.xlf.lcl:12330]: FYI, you don't need to update any of the LCL/XLF files. That will be done automatically by Alex when he runs his localization scripts. So I'd just revert that to prevent any possible weirdness that manually updating them might cause.

Comment (aasimkhan30) [reply resources/localization/LCL/de/sql.xlf.lcl]: I was under the assumption that we must delete strings when we change the text with the same localization key.

Comment (Charles-Gagnon) [reply resources/localization/LCL/de/sql.xlf.lcl]: So technically if we want to be really on top of things you *should* change the loc key (essentially "deleting" the old key and adding a new one with a different name). But generally we don't bother doing this - as the normal localization process should ensure that the strings are kept all up to date. Which as a side note - you didn't do, since you just updated the text in the string but not the key itself. 

There is a small risk of having a string that was localized previously not get updated in time, thus for non-EN languages showing the wrong content. But we haven't gotten a complaint about that yet so it's not something we've bothered to do so far

**Review by Charles-Gagnon — APPROVED** [review]
Get approval from Karl and/or Alan before merging this in to make sure they agree with my thoughts here about the preview state of the feature.

**Review by alanrenmsft — APPROVED** [review]

**Review by kburtram — APPROVED** [review]

**Pull request merged**
